database/migrations/2021_12_10_100000_create_membership_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Facades\DB;
+
+class CreateMembershipTable extends Migration
+{
+  /**
+   * Run the migrations.
+   *
+   * @return void
+   */
+  public function up()
+  {
+    DB::statement("CREATE TABLE `membership` (
```

Comment:
Indentation is not looking better, right now.

Reply:
fixed

Reply:
it is still in same state. I recommend to take a look on my SQL code in the lesson.